tests/require-arrow.test.js

```diff
@@ -0,0 +1,136 @@
+'use strict';
+
+describe('Plugin for require blocks', () => {
+  it('transforms require blocks with one dependency', () => {
+    expect(`
+      require(['llamas'], (llama) => {
+        llama.doSomeStuff();
+      });
+    `).toBeTransformedTo(`
+      (() => {
+        var llama = require('llamas');
+        llama.doSomeStuff();
+      })();
+    `);
+  });
+
+  it('transforms require blocks with one dependency and implicit return', () => {
+    expect(`
+      require(['llamas'], (llama) => llama.doSomeStuff());
+    `).toBeTransformedTo(`
+      (() => {
+            var llama = require('llamas');
+            return llama.doSomeStuff();
+      })();
+    `);
+  });
+
+  it('transforms require blocks with multiple dependencies', () => {
+    expect(`
+      require(['llamas', 'frogs'], (llama, frog) => {
+        llama.doSomeStuff();
+        frog.sayRibbit();
+      });
+    `).toBeTransformedTo(`
+      (() => {
+        var llama = require('llamas');
+        var frog = require('frogs');
+        llama.doSomeStuff();
+        frog.sayRibbit();
+      })();
+    `);
+  });
+
+  it('transforms require blocks with multiple dependencies and implicit return', () => {
+    expect(`
+      require(['llamas', 'frogs'], (llama, frog) => llama.doSomeStuff(frogs));
```

Comment:
Let's use consistent variable naming here i.e. frog -> frogs. It doesn't really matter for the tests but I think it makes sense to have them represent actual programs as closely as is reasonable.